nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -378,6 +397,36 @@ void CollisionMonitor::process(const Velocity & cmd_vel_in)
     }
   }
 
+  if (collision_points_marker_pub_) {
+    // visualize collision points with markers
+    visualization_msgs::msg::MarkerArray marker_array;
+    if (collision_points.size() > 0) {
```

Comment:
I'd remove this so that an empty message is published when empty so that we clear out old stuff